packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="info">
+    <template v-slot:header>
+      <div class="context__header">Navie is aware of your project</div>
```

Comment:
I think we should iterate a bit on the "is aware" / "is not aware" message. It doesn't quite seem evident to me.

Reply:
I changed it based on our conversation in Slack and updated the screenshots.